client/src/routes/Menu/menu.module.css

```diff
@@ -1,7 +1,35 @@
+.imgWrapper {
+  position: absolute;
+  top:10%;  
+  left: 50%; 
+  transform: translate(-50%, -50%)
+}
+
+.img {
+  width: 75px;
+}
+
+.companyName{
+  font-family: Raleway, sans-serif;
+  color:rgba(255, 255, 255, 0.90);
+  font-style: italic;
+  font-size: var(--h0mobile);
+  font-weight: var(--font-weight-extrabold);
+  text-shadow: 4px 4px 5px rgba(0, 0, 0, 0.5);
+  text-align: left;
+}
+
+.companyWrapper{
+  position: absolute;
+  bottom:0;  
+  left: 50%; 
+  transform: translate(-50%, -50%)
+}
+
 .menu {
-  font-size: 12px;
+  font-size: var(--h4mobile);
   font-family: "Raleway";
-  font-weight: 600;
+  font-weight: var(--font-weight-normal);
   color: var(--white);
```

Comment:
In line 42, class navigationContainer with position relative does not do anything. It could be deleted.

Reply:
navigation container removed. 

Company name position changed.

naviggationContainerA removed.